data/json/hobbies.json

```diff
@@ -0,0 +1,93 @@
+    "points": -1,
+    "addictions": [ { "intensity": 10, "type": "nicotine" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "crackhead",
+    "name": "Crackhead",
+    "description": "Cocaine.  It is, indeed, a helluva drug.  You blew your money on some dust, and before you knew it you were turning tricks behind the local CVS just to score one more line.  Where are you going to get your next fix now?",
+    "points": -1,
+    "addictions": [ { "intensity": 20, "type": "crack" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "tweaker",
+    "name": "Tweaker",
+    "description": "You're not entirely sure what happened last night, but you woke up on the floor and everything has gone completely to shit.  The only thing running through your head, though, is where you're gonna find your next hit.",
+    "points": -2,
+    "addictions": [ { "intensity": 30, "type": "amphetamine" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "pillhead",
+    "name": "Pillhead",
+    "description": "After an accident in your youth, you got addicted to the opiates treating your pain.  With the pharmacies shut down and your dealers turned undead, satisfying those cravings just got a lot more difficult.",
+    "points": -1,
+    "addictions": [ { "intensity": 20, "type": "opiate" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "alcoholic",
+    "name": "Alcoholic",
+    "description": "You found solace in the bottom of a bottle.  God-damn, you need a drink.",
+    "points": -1,
+    "addictions": [ { "intensity": 20, "type": "alcohol" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "cosplay",
+    "name": "Cosplay",
+    "description": "You liked to make your own costumes and wore them to various conventions.  Now that everyone is dead or dying, maybe this will be useful for something else.",
+    "points": 1,
+    "skills": [ { "level": 1, "name": "tailor" } ],
+    "proficiencies": [ "prof_closures", "prof_leatherworking_basic" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "lockpicking",
+    "name": "Lockpicking",
+    "description": "Silently picking people's locks while they were away was a rush for you.  Will you still feel the same thrill now that no one is left to stop you?",
+    "points": 1,
+    "proficiencies": [ "prof_lockpicking" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "parkour",
+    "name": "Parkour",
+    "description": "You've practiced parkour for many years, and made the world your playground.  It wouldn't be a lie to say that running is your life.  Which is good, because now that the end has come, you're running for your life.",
+    "points": 2,
+    "traits": [ "PARKOUR" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "camping",
```

Comment:
Might be worth refluffing this as a bit more wilderness survival-y given the skills